R/scale-viridis.r

```diff
@@ -0,0 +1,68 @@
+#' Viridis colour scales from viridisLite
+#'
+#' @description
+#' The `viridis` scales provide color maps that are perceptually uniform in both
+#' color and black-and-white. They are also designed to be perceived by viewers
+#' with common forms of color blindness. See also
+#' \url{https://bids.github.io/colormap/}.
+#' 
+#' @inheritParams viridisLite::viridis
+#' @inheritParams scales::gradient_n_pal
+#' @inheritParams continuous_scale
+#' @param ... Other arguments passed on to [discrete_scale()] or
+#' [continuous_scale()] to control name, limits, breaks, labels and so forth.
+#' @family colour scales
+#' @rdname scale_viridis
+#' @export
+scale_colour_viridis_discrete <- function(..., alpha = 1, begin = 0, end = 1,
+                                          direction = 1, option = "D") {
+  discrete_scale("colour", "viridis_discrete",
```

Comment:
Can you please use tidyverse style here: http://style.tidyverse.org/syntax.html#long-lines